frontend/src/components/Forms/Sheet/Sheet.module.scss

```diff
@@ -10,7 +10,17 @@
   justify-content: center;
 }
 
-.sheet {
+.containerGamemode {
+  height: 107vh;
+  width: 100vw;
+  opacity: 95%;
+  background: rgba(0, 0, 0, 0.2);
```

Comment:
using this in conjunction with `opacity` is a bit confusing

Reply:
background just darkens the pano a bit